draft-ietf-ppm-dap.md

```diff
@@ -1060,69 +1060,149 @@ of relevant: issue #89]]
-
-~~~
-Leader                                                 Helper
-
-Aggregate request (Reports 1-10, Job = N) --------------->  \
-<----------------------------- Aggregate response (Job = N) | Reports
-Aggregate request (continued, Job = N) ------------------>  | 1-10
-<----------------------------- Aggregate response (Job = N) /
-
-
-Aggregate request (Reports 11-20, Job = M) -------------->  \
-<----------------------------- Aggregate response (Job = M) | Reports
-Aggregate request (continued, Job = M) ------------------>  | 11-20
-<----------------------------- Aggregate response (Job = M) /
-~~~
-{: #aggregation-flow-illustration title="Aggregation Flow (batch size=20).
-Multiple aggregation flows can be executed at the same time."}
-
-The aggregation flow can be thought of as having three phases for transforming
-each valid input report share into an output share:
-
-- Initialization: Begin the aggregation flow by sharing report shares with each
-  Helper. Each Aggregator, including the Leader, initializes the underlying VDAF
-  instance using these report shares and the VDAF configured for the
-  corresponding measurement task.
-- Continuation: Continue the aggregation flow by exchanging messages produced by
-  the underlying VDAF instance until aggregation completes or an error occurs.
-  These messages do not replay the shares.
+begin the process of validating and aggregating them with the Helper. To enable
+the system to handle large batches of reports, this process can be parallelized
+across many "aggregation jobs" in which small subsets of the reports are
+processed independently. Each aggregation job is associated with exactly one
+DAP task, but a task can have many aggregation jobs.
+
+The primary objective of an aggregation job is to run the VDAF preparation
+process described in {{!VDAF, Section 5.2}} for each report in the job.
+Preparation has two purposes:
+
+1. To "refine" the input shares into "output shares" that have the desired
+   aggregatable form. For some VDAFs, like Prio3, the mapping from input to
+   output shares is some fixed, linear operation, but in general the mapping is
+   controlled dynamically by the Collector and can be highly non-linear. In
+   Poplar1, for example, the refinement process involves a sequence of
+   "candidate prefixes" and the output consists of a sequence of 0s and 1s,
+   each indicating whether the corresponding candidate is a prefix of the
+   measurement from which the input shares were generated.
+
+1. To verify that the output shares, when combined, correspond to a valid,
+   refined measurement, where validity is determined by the VDAF itself. For
+   example, the Prio3Sum variant of Prio3 ({{Section 7.4.2 of !VDAF}}) requires
+   that the output shares sum up to an integer in a specific range; for
+   Poplar1, the output shares are required to sum up to a one-hot vector (i.e.,
+   a vector that is non-zero in at most one position).
```

Comment:
nit:
```suggestion
   Poplar1, the output shares are required to sum up to a vector that is non-zero in at most one position.
```
We don't need to define the term "one-hot" unless we're going to use it elsewhere in this document.